sig-apps/annual-report-2021.md

```diff
@@ -4,134 +4,120 @@
+     - [1591 - Allow DaemonSets to surge during update like Deployments](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/1591-daemonset-surge/README.md) - 1.21
+     - [2214 - Indexed Job](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2214-indexed-job/README.md) - 1.21
+     - [2232 - Suspend Job](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2232-suspend-jobs/README.md) - 1.21
+     - [2255 - ReplicaSet Pod Deletion Cost](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2255-pod-cost/README.md) - 1.21
+     - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md) - 1.22
+     - [2599 - minReadySeconds for StatefulSets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md) - 1.22
+     - [1847 - Auto delete PVCs created by StatefulSet](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md) - 1.23
+     - [2879 - Track ready Pods in Job status](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2879-ready-pods-job-status/README.md) - 1.23
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Growing additional reviewers and approvers, also see no. 4 below.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Open untriaged issues and PRs
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - It's up-to-date.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We’re working with SIG-CLI on starting a monthly review club, details to be announced soon.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - [11 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Apps&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1609455600000&to=1639350000000) contributed code in 2021.
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - Increase the test coverage for all controllers.
```

Comment:
Is there one or more issues that cover this? When we consolidate down reports to be sent out, it'd be great if there was a follow up item we could direct people to

Reply:
There are two efforts which are happening in parallel, conformance testing for all apps API groups and unit test coverage. But there is no single issue tracking that, it's mostly scattered around several :/